vars/common-vars.yml

```diff
@@ -104,11 +104,11 @@ suse_airship_components_image_tag: |
                                    %}
                                    {% else %}
                                    {% set _img_tag = {
-                                     'armada': '1110c23cd192ea5eff77629d10924b290140d6e2-opensuse_15',
-                                     'shipyard': 'cd6f154da56079ba408b922ed748a80b622c3592-opensuse_15',
-                                     'airflow': 'cd6f154da56079ba408b922ed748a80b622c3592-opensuse_15',
-                                     'deckhand': 'd8c8e15074d484c356c03460c0acbaea96705150-opensuse_15',
-                                     'pegleg': '652c3abefd0f647cfc45096aa76cb76bfaf8a805-opensuse_15'
+                                     'armada': '46bb9bd8b35905294920eb4fdf30593cb4899780-opensuse_15',
+                                     'shipyard': 'a9ff859df980fbb95a912fc49312ef3c51f65b3f-opensuse_15',
+                                     'airflow': 'a9ff859df980fbb95a912fc49312ef3c51f65b3f-opensuse_15',
+                                     'deckhand': '971c901c4da36e3e31be58d4be8b0e194e715d8f-opensuse_15',
+                                     'pegleg': '70f2db465277b60d80e4bcbf347be565b6f81a05-opensuse_15'
```

Comment:
`opensuse_15` shouldnt it be 15.1? does the upstream airship images only point to 15 with no point release? seems like a problem waiting to happen :(

Reply:
In upstream, we wanted to keep this specific to a major release from maintenance perspective. The reason being, we want to maintain one version of Dockerfile per major release of distro. There is not much changed in (terms of package dependency, executable paths/ env etc) within minor releases of a major version.